docs/digger/installation/digger-install-intro.asciidoc

```diff
@@ -14,4 +14,8 @@ It is made of following components:
 * A suite of sample Jenkinsfiles to demonstrate how to build mobile applications using Jenkins pipeline
 
 This project is designed to be used with the RedHat Application Platform (RHMAP). 
```

Comment:
> RedHat Application Platform (RHMAP) 

Red Hat Mobile Application Platform (RHMAP) :) 
Can you fix this while you're at it?